app/mailers/application_mailer.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  default from: 'anna.tytko@monterail.com'
```

Comment:
Consider extraction to ENV variables. We trying to not persist in code real email addresses, phone number etc.